frappe/translate.py

```diff
@@ -117,6 +117,12 @@ def get_dict(fortype, name=None):
 			messages += frappe.db.sql("select 'Role:', name from tabRole")
 			messages += frappe.db.sql("select 'Module:', name from `tabModule Def`")
 
+			# Adding module labels
+			messages.append(('sites/assets/js/desk.min.js', 'Modules'))
```

Comment:
Had to do this because the category names could not be fetched from desk.js
Maybe if we fetch. keywords from `config.py` files this will not have to be hardcoded

Reply:
Found a better solution